reporter/src/main/kotlin/model/AmazonOssAttributionBuilderService.kt

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
I'd rather put this class in the `utils` package, `model` is currently only used for the evaluated model.

Reply:
But doesn't `model` still fit better, as these *are* pure data model classes after all? `utils` sounds more like utility *functions*, IMO. Another alternative would be to move this to `reporters` directly.

Reply:
I would argue it's an external service and therefore better fits in the `utils` package. To me `utils` also sounds like utility *classes* or *services*.

Reply:
Well, ok, let's go for `utils` then, @julscha.

Reply:
done :)